content/en/docs/Configuration/authentication/anonymous.md

```diff
@@ -2,6 +2,7 @@
 title: "Anonymous strategy"
 linktitle: "Anonymous"
 description: "Access Kiali with no authentication."
+weight: 10
```

Comment:
I had removed these weights and just defaulted to alphabetical, do we need a non-default ordering?  For me it's OK if Session Options is sorted alphabetically, I'm fine leaving them here, though, if you feel it should be at the bottom.

Reply:
It's almost alphabetical. I added weights only to place the session configs at the end.
I find weird that _Session Options_ end in the middle of the auth strategy pages in the navigation menu.